setup.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, mdsa development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# this setup.py file is heavlily based on scikit-bio's setup.py file
+
+import re
+import ast
+
+from setuptools import find_packages, setup
+
+# https://github.com/mitsuhiko/flask/blob/master/setup.py
+_version_re = re.compile(r'__version__\s+=\s+(.*)')
+
+with open('mdsa/__init__.py', 'rb') as f:
+    hit = _version_re.search(f.read().decode('utf-8')).group(1)
+    version = str(ast.literal_eval(hit))
+
+classes = """
+    Development Status :: 1 - Planning
+    License :: OSI Approved :: BSD License
+    Topic :: Software Development :: Libraries
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Bio-Informatics
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
```

Comment:
@mortonjt, I don't recall that conversation, my bad. We can do 3-only
then. Anyone has objections?

On (Oct-19-15|16:29), Antonio Gonzalez wrote:
> > +# https://github.com/mitsuhiko/flask/blob/master/setup.py
> > +_version_re = re.compile(r'__version__\s+=\s+(.*)')
> > +
> > +with open('mdsa/__init__.py', 'rb') as f:
> > +    hit = _version_re.search(f.read().decode('utf-8')).group(1)
> > +    version = str(ast.literal_eval(hit))
> > +
> > +classes = """
> > +    Development Status :: 1 - Planning
> > +    License :: OSI Approved :: BSD License
> > +    Topic :: Software Development :: Libraries
> > +    Topic :: Scientific/Engineering
> > +    Topic :: Scientific/Engineering :: Bio-Informatics
> > +    Programming Language :: Python
> > +    Programming Language :: Python :: 2
> > +    Programming Language :: Python :: 2.7
> 
> Well, ideally we will support both but perhaps for simplicity we should focus on 3 ... 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/mds-approximations/pull/7/files#r42439208